platform/ios/src/UIViewController+MGLAdditions.h

```diff
@@ -0,0 +1,11 @@
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface UIViewController (MGLAdditions)
+
++ (UIViewController *)mgl_topMostViewControllerInHierarchyWith:(UIViewController *)viewController;
```

Comment:
I would also prefer this to be an instance method to support being able to call it like in https://github.com/mapbox/mapbox-gl-native/pull/8837#discussion_r113830166. Otherwise I think this is 👍